__tests__/endpoints.test.js

```diff
@@ -237,4 +238,74 @@ describe('GET /api/articles/:article_id/comments', () => {
         expect(response.body.msg).toBe('article does not exist');
         });
     });
-});
\ No newline at end of file
+});
+
+describe('POST /api/articles/:article_id/comments', () => {
+    test('responds with a 201 status code', () => {
```

Comment:
we dont really need this one as we are doing the same thing in the following test - can juts say 201: test description in the next test